### PR TITLE
Switch to using Foreman client repository for Katello

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by jturel** [comment]
LGTM @ehelms. Have the new client repos been tested at all?

File: plugins/katello/nightly/installation/clients.md [hunk 1]
```diff
@@ -11,66 +11,65 @@ Client machines can be added in one of two ways: manually or via a provisioned h
...
      <option value="sles11">Suse Enterprise Linux Server 11</option>
      <option value="sles12">Suse Enterprise Linux Server 12</option>
   </select>
 </p>
 
 <div id="el5" markdown="1">
 {% highlight bash %}
-yum install -y https://fedorapeople.org/groups/katello/releases/yum/{{ page.version }}/client/el5/x86_64/katello-client-repos-latest.rpm
 yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-5.noarch.rpm
+yum install -y http://yum.theforeman.org/client/{{ page.foreman_version }}/el5/x86_64/foreman-client-release.rpm
```
Comment (ekohl) [inline plugins/katello/nightly/installation/clients.md]: We do have https available and for release RPMs that's strongly preferred IMHO

**Review by ekohl — CHANGES_REQUESTED** [review]

File: plugins/katello/nightly/installation/clients.md [hunk 2]
```diff
@@ -11,66 +11,65 @@ Client machines can be added in one of two ways: manually or via a provisioned h
...
 {% endhighlight %}
 </div>
 
 <div id="el6" style="display:none;" markdown="1">
 {% highlight bash %}
-wget https://copr.fedoraproject.org/coprs/dgoodwin/subscription-manager/repo/epel-6/dgoodwin-subscription-manager-epel-6.repo -O /etc/yum.repos.d/dgoodwin-subscription-manager-epel-6.repo
 yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm
-yum install -y https://fedorapeople.org/groups/katello/releases/yum/{{ page.version }}/client/el6/x86_64/katello-client-repos-latest.rpm
+yum install -y http://yum.theforeman.org/client/{{ page.foreman_version }}/el6/x86_64/foreman-client-release.rpm
```
Comment (ik5) [inline plugins/katello/nightly/installation/clients.md]: what does `page.foreman_version` holds? I went to the path and there is only `nightly` there.

and as @ekohl wrote, please prefer `https` ;)

Comment (ehelms) [reply plugins/katello/nightly/installation/clients.md]: The page template defines `page.foreman_version` depending on the matchup of releases. So for Katello nightly this value is nightly

**Comment by ehelms** [comment]
Updated to https

**Comment by jturel** [comment]
I tested the client repos for el7 and f27 - all packages installed and behave correctly. ACK from me

**Review by jturel — APPROVED** [review]

**Review by ekohl — APPROVED** [review]

**Pull request merged**